exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -86,7 +86,20 @@ object SchemaUtils {
         } + alters
     }
 
-    fun createSequence(name: String) = Seq(name).createStatement()
+    fun createSequence(vararg seq: Sequence, inBatch: Boolean = false) {
+        with(TransactionManager.current()) {
+            val createStatements = seq.flatMap { it.createStatement() }
+            execStatements(inBatch, createStatements)
+            commit()
```

Comment:
I'll remove it,
 I dont think that it is required. In any DDL statement there is an implicit commit.
I put it because it's used in create table method (org.jetbrains.exposed.sql.SchemaUtils.create).
So in that method, is it required?